### PR TITLE
Upgrade to ginkgo v2 in tests module

### PR DESCRIPTION
**Pull request by rroberts2222**
# Description

Upgrade to ginkgo v2 in tests module to address this issue: https://github.com/cloudfoundry/windows-syslog-release/issues/22

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Testing performed?

- [X] Unit tests
- [ ] Integration tests
- [x] Acceptance tests

## Checklist:

- [X] This PR is being made against the `main` branch, or relevant version branch
- [ ] I have made corresponding changes to the documentation
- [ ] I have added testing for my changes

If you have any questions, or want to get attention for a PR or issue please reach out on the [#logging-and-metrics channel in the cloudfoundry slack](https://cloudfoundry.slack.com/archives/CUW93AF3M)

### CODE REVIEW

**Review by ctlong — CHANGES_REQUESTED** [review]
This PR shouldn't change the `src/blackbox` ref. Our CI will bump that git module for us.

Can you please remove that change from your commit, or this PR.

File: scripts/test [hunk 1]
```diff
@@ -11,6 +11,6 @@ bosh upload-release https://bosh.io/d/github.com/cloudfoundry/syslog-release
 bosh upload-release https://bosh.io/d/github.com/cloudfoundry-incubator/windows-utilities-release
 
 pushd "$(dirname "$0")/../tests"
-  go install github.com/onsi/ginkgo/ginkgo@latest
+  go install github.com/onsi/ginkgo/v2/ginkgo@latest
   ginkgo -r -nodes=2 "$@"
```
Comment (ctlong) [inline scripts/test:15]: The ginkgo v2 CLI prefers `--procs=2`, as `-nodes` is deprecated.

Comment (rroberts2222) [reply scripts/test]: Replaced -nodes with --procs, thank you!

**Review by ctlong — APPROVED** [review]

**Comment by ctlong** [comment]
LGTM! I'll merge it if it passes the PR pipeline. Thanks @rroberts2222!

**Pull request merged**